source/code/plugin/ApplicationInsightsUtility.rb

```diff
@@ -21,10 +22,12 @@ class ApplicationInsightsUtility
   @@EnvApplicationInsightsEndpoint = "APPLICATIONINSIGHTS_ENDPOINT"
   @@EnvControllerType = "CONTROLLER_TYPE"
   @@EnvContainerRuntime = "CONTAINER_RUNTIME"
+  @DefaultAppInsightsEndpoint = "https://dc.services.visualstudio.com/v2/track"
```

Comment:
do we need to hard-code the AI ingestion endpoint here ? I see that the API that requires proxy to be passed needs it, but this endpoint is already in the appinsights sdk that can be reused, rather then being defined yet again